mobile/src/navigation/app/components/drawer-item/drawer-item.tsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigation } from '@react-navigation/native';
+import React, { FC } from 'react';
+import { Pressable } from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+import { AppColor } from '~/common/enums/enums';
+import { DrawerNavigationItem } from '~/common/types/types';
+import { Text } from '~/components/common/common';
+
+import { styles } from './styles';
+
+const CustomDrawerItem: FC<DrawerNavigationItem> = ({
+  name,
+  icon,
+  focusedRouteName,
+}) => {
+  const navigation = useNavigation();
+  const focused = focusedRouteName === name ? true : false;
```

Comment:
`focusedRouteName === name` already return `true` if it's truthy and `false` if it's falsy,, so ternary usage is redundant
```suggestion
  const isFocused = focusedRouteName === name;
```